src/Features/Core/Portable/AddImport/CodeActions/InstallPackageAndAddImportCodeAction.cs

```diff
@@ -50,7 +50,7 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputePreviewOp
             {
                 // Make a SolutionChangeAction.  This way we can let it generate the diff
                 // preview appropriately.
-                var solutionChangeAction = new SolutionChangeAction("", c => GetUpdatedSolutionAsync(c));
+                var solutionChangeAction = new SolutionChangeAction("", c => GetUpdatedSolutionAsync(c), "");
```

Comment:
Not sure why the title is empty string here. Is this code action (for some reason) non user-facing?

Reply:
yes.  it is only used to get the operations out.